common/src/IO/MdlParser.cpp

```diff
@@ -229,8 +230,11 @@ namespace TrenchBroom {
             const size_t skinVertexCount = readSize<int32_t>(cursor);
             const size_t skinTriangleCount = readSize<int32_t>(cursor);
             const size_t frameCount = readSize<int32_t>(cursor);
+            [[maybe_unused]]
+            const size_t syncType = readSize<int32_t>(cursor);
```

Comment:
I usually just comment out the variable declaration, i.e. `/* const size_t syncType = */ readSize...`